4.8/Dockerfile

```diff
@@ -40,7 +39,7 @@ RUN ARCH= && dpkgArch="$(dpkg --print-architecture)" \
   && rm "node-v$NODE_VERSION-linux-$ARCH.tar.xz" SHASUMS256.txt.asc SHASUMS256.txt \
   && ln -s /usr/local/bin/node /usr/local/bin/nodejs
 
-ENV YARN_VERSION 0.24.4
+ENV YARN_VERSION 1.1.0
```

Comment:
Should the Yarn change with this?

Reply:
Ah, yeah, that should not be updated (the update.sh script is probably to blame here :)

Reply:
Yep it is....will fix that